chromium_src/chrome/browser/ui/views/permissions/permission_prompt_bubble_view.cc

```diff
@@ -12,6 +12,7 @@
 #include "brave/components/permissions/permission_lifetime_utils.h"
 #include "brave/grit/brave_generated_resources.h"
 #include "chrome/browser/ui/browser_tabstrip.h"
+#include "chrome/browser/ui/views/chrome_layout_provider.h"
```

Comment:
is it because widevine doesn't exist for linux arm64? If so, please also move everything except widevine_permission_request.h outside the buildflag as they are all improperly placed inside it